.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ executors:
 # COMMANDS
 # ------------------
 commands:
-  publish_artefact:
+  publish_artefact_rest_client:
```

Comment:
Is there any chance we can combine the publish commands? We're already passing the release type, so why not pass the artefact/module name and consolidate them.

Reply:
I haven't tried passing parameter from circleci jobs to commands. Will test and let you know.